nfsstat/datadog_checks/nfsstat/nfsstat.py

```diff
@@ -38,8 +38,10 @@ def check(self, instance):
         all_devices = []
         this_device = []
         custom_tags = instance.get("tags", [])
-
         for l in stat_out.splitlines():
+            if l.find(b'No NFS mount point') >= 0:
```

Comment:
Let's maybe only check this for the first line?

Reply:
Having it in the for loop also covers the case if there is a newline and then this. I think it's safe to keep it here as it should be the first non empty line returned in the output.

Reply:
Looks like its indeed always the first line. We can pull it out at the beginning.